### PR TITLE
documentation initial setup

### LINKED ISSUE #3: Add documentation website
Use Nuxt docs or vitepress.

First content:
- content from readme file.
- content from node sdk readme file

### PR DESCRIPTION
**Pull request by Himasnhu-AT**
Resolve #3 , adding the documentation website using vitepress.

Configuration:
```
{
    name: "Prici",
    description: "Prici is an open-source project to manage plans and pricing for any SaaS application.",
    navbar: {
        Home, 
        Github
    },
    Sidebar: {
        Introduction: {
            readme file of root as introduction, 
            readme file of sdk as skd
        }
    }
}
```

### CODE REVIEW

File: documentation/sdk.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### Create new AccountPlan
+
+An account plan is the actual state of a plan for a specific account
+
+```typescript
+
+const demoAccount = await sdk.AccountPlan.insert({
+  accountId: 'demo-account',
+  planId: basicPlan.id,
```
Comment (davidmeirlevy) [inline documentation/sdk.md:78]: This interface was changed recently.
Look at the currrent sdk doc.

**Review by davidmeirlevy — COMMENTED** [review]
Great work! Thanks!
Just notice the comment about planId.

File: documentation/sdk.md (hunk 1 above)
Comment (Himasnhu-AT) [reply documentation/sdk.md:78]: update from init-data.ts from demo backend server

Comment (davidmeirlevy) [reply documentation/sdk.md]: The field doesn't call planId.
It's just "plan", and you can put the entire plan object.

**Review by davidmeirlevy — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
Thanks @Himasnhu-AT !

**Comment by Himasnhu-AT** [comment]
Happy to help

**Pull request merged**